docs/docs/overview/clouds.md

```diff
@@ -28,11 +28,19 @@ With its [CVM offering](https://docs.microsoft.com/en-us/azure/confidential-comp
 
```

Comment:
do you think we should keep the line above regarding customizable firmware on Azure for now or remove?

Reply:
I'd keep it for now.